docs/en/guides/operational/gitops-with-amg/gitops-with-amg.md

```diff
@@ -0,0 +1,65 @@
+# Using GitOps and Grafana Operator with Amazon Managed Grafana
+
+## How to use this guide
+
+This Observability best practices guide is meant for developers and architects who want to understand how to use [grafana-operator](https://github.com/grafana-operator/grafana-operator#:~:text=The%20grafana%2Doperator%20is%20a,an%20easy%20and%20scalable%20way.) as a Kubernetes operator on your Amazon EKS cluster to create and manage the lifecycle of data sources such as Amazon Managed Service for Prometheus, Amazon CloudWatch, AWS X-Ray to Amazon Managed Grafana and create Grafana dashboards in Amazon Managed Grafana in a Kubernetes native way.
```

Comment:
Manage the lifecycle of "Amazon Managed Service for Prometheus, Amazon CloudWatch, AWS X-Ray". Is it? I thought this was only grafana resources? What does grafana-operator actually do in managing these other resources' lifecycles?